app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mutable_variables = [my_favourite_films, marks, collection_of_coins]
```

Comment:
create only one variable `sorted_variables` and put the data there


example:
```python
a = 123
b = []
c = "Hi!"
d = [1, 2]

sorted_variables = {
    "mutable": [b, d],
    "immutable": [a, c]
}
```